malspam_fake_shipping_notification_free_file_host.yml

```
@@ -0,0 +1,31 @@
+name: "Malspam: Fake Shipping notification with link to free file hosting Email"
```

```suggestion
name: "Fake shipping notification with link to free file hosting"
```
we might put `Malspam`(or just Spam) as a Tag instead? defer to you